repos/base-linux/src/core/include/irq_object.h

```diff
@@ -0,0 +1,38 @@
+
+#ifndef _IRQ_OBJECT_H_
+#define _IRQ_OBJECT_H_
+
+#include <base/thread.h>
+
+namespace Genode
+{
+    class Irq_object;
+};
+
+class Genode::Irq_object : public Thread_deprecated<4096>
+{
+
+    private:
+
+        Genode::Signal_context_capability _sig_cap;
+        Genode::Lock _sync_ack;
+        Genode::Lock _sync_bootup;
+        unsigned _irq;
+        int _fd;
```

Comment:
`_irq` and `_fd` should better be const

Reply:
`_fd` cannot be const since it needs to be set in `Genode::Irq_object::_associate()`.

Reply:
alright. Thanks for clarifying.